### PR TITLE
verify-libs-and-licenses

### PR DESCRIPTION
**Pull request by vieiro**
Getting rid of those tiny whitespace errors.
Yes, vim rocks.

### CODE REVIEW

File: html.parser/external/htmlparser-1.2.1-license.txt [hunk 1]
```diff
@@ -1,83 +1,83 @@
 Name: HTML 5 Parser Patched
-Description: HTML4/5 parser patched for editing purposes
+Description: HTML4/5 parser patched for editing purpose
```
Comment (steffendietz) [inline html.parser/external/htmlparser-1.2.1-license.txt]: I think, your script removed some letters by accident.

File: html.parser/external/htmlparser-1.2.1-license.txt [hunk 2]
```diff
@@ -1,83 +1,83 @@
...
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
 The following license applies to the Live DOM Viewer:
 
-Copyright (c) 2000, 2006, 2008 Ian Hickson and various contributors
+Copyright (c) 2000, 2006, 2008 Ian Hickson and various contributor
```
Comment (steffendietz) [inline html.parser/external/htmlparser-1.2.1-license.txt]: Same here.

File: html.parser/external/htmlparser-1.2.1-license.txt [hunk 3]
```diff
@@ -1,83 +1,83 @@
...
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
+in the Software without restriction, including without limitation the right
```
Comment (steffendietz) [inline html.parser/external/htmlparser-1.2.1-license.txt]: Here.

File: html.parser/external/htmlparser-1.2.1-license.txt [hunk 4]
```diff
@@ -1,83 +1,83 @@
...
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit persons to whom the Software i
```
Comment (steffendietz) [inline html.parser/external/htmlparser-1.2.1-license.txt]: Also here.

**Comment by vieiro** [comment]
Thanks, Steffen! I mispelled a regex, I think.

**Comment by geertjanw** [comment]
Looks good, thanks, and merging.

**Pull request merged**